async-nats/src/jetstream/mod.rs

```diff
@@ -24,3 +25,17 @@ pub use context::Context;
 pub fn new(client: Client) -> Context {
     Context::new(client)
 }
+
+#[derive(Debug)]
+pub struct JetStreamMessage {
```

Comment:
Could probably lean into this being tuple type that HOLDS a message and context naming wise, something something pair 🤔

TBD.

Reply:
hm. I think it's a little easier to use as a struct, but no hard opinions here. TBD.

Reply:
I would stick to struct with deref.